yjit/src/cruby_bindings.inc.rs

```diff
@@ -1126,8 +1126,8 @@ extern "C" {
     pub fn rb_attr_get(obj: VALUE, name: ID) -> VALUE;
     pub fn rb_obj_info_dump(obj: VALUE);
     pub fn rb_reg_new_ary(ary: VALUE, options: ::std::os::raw::c_int) -> VALUE;
-    pub fn rb_class_allocate_instance(klass: VALUE) -> VALUE;
     pub fn rb_obj_info(obj: VALUE) -> *const ::std::os::raw::c_char;
+    pub fn rb_class_allocate_instance(klass: VALUE) -> VALUE;
     pub fn rb_shape_id_offset() -> i32;
```

Comment:
Why was this changed?

Reply:
because the yjit github actions build complained at me and told me to apply the diff